nncf/experimental/tensorflow/nncf_operation.py

```diff
@@ -0,0 +1,40 @@
+"""
+ Copyright (c) 2021 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from nncf.tensorflow.layers.operation import NNCFOperation
+from nncf.experimental.tensorflow.nncf_network import NNCFNetwork
+
+
+class NNCFOperationV2(NNCFOperation):
```

Comment:
Same comment about V2

Reply:
Thanks for the comment! I added the additional explanation to the docstring.